certs/certs.go

```diff
@@ -0,0 +1,67 @@
+// Package certs handles persisting and loading certs
+package certs
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"github.com/levenlabs/locksmith/config"
+	"log"
+	"os"
+	"time"
+)
+
+func init() {
+	if config.CertsFile == "" {
+		return
+	}
+	// make sure we can write to the file
+	if _, err := os.Stat(config.CertsFile); err != nil {
+		if !os.IsNotExist(err) {
+			log.Fatalf("Failed to read certs file: %s", err)
+		}
+		//create the file since it doesn't exist
+		f, err := os.Create(config.CertsFile)
+		if err != nil {
+			log.Fatalf("Failed to create certs file: %s", err)
+		}
+		defer f.Close()
+
+		err = f.Chmod(0600)
+		if err != nil {
+			log.Fatalf("Failed to chmod certs file: %s", err)
+		}
+	}
+}
+
+func SaveCert(hostname, remoteAddr, cert string) error {
```

Comment:
This function needs to have some kind of lock or channel around it, right now it could be called more than once at the same time and that could create some weirdness with the opening of the file.